starter-code/index.html

```diff
@@ -0,0 +1,250 @@
+                    <li class="nav-item mr-2">Blog</li>
+                    <li class="nav-item mr-2">Parent Page>></li>
+                    <li class="nav-item mr-2">Contact</li>
+                </ul>
+            </div>
+            </div>
+        </nav>
+        <!-- end row -->
+    </header>
+    <div id="slidebox" class="carousel slide" data-ride="carousel">
+            
+
+        <div class="carousel-inner">
+            <div class="carousel-item active">
+                <img class="d-block w-100" src="img/slider.jpg" alt="First slide">
+                <div class="overlay" style="background-image: url('img/overlay.png');height: 100%;position: absolute; top:0;width: 100%;z-index: 2;"></div>
+                <div class="carousel-caption d-none d-md-block" style="z-index: 5">
+                    <h3>#YOLO</h3>
+                    <p>Alea Jacta est comme dirait l'autre !</p>
+                </div>
+            </div>
+            <div class="carousel-item">
+                <img class="d-block w-100" src="img/slider2.jpg" alt="Second slide">
+                <div class="overlay" style="background-image: url('img/overlay.png');height: 100%;position: absolute; top:0;width: 100%;z-index: 2;"></div>
+                <div class="carousel-caption d-none d-md-block" style="z-index: 5">
+                    <h3>SINE QUA NON</h3>
+                    <p>TU QUOQUI FILI !</p>
+                </div>
+            </div>
+            <div class="carousel-item">
+                <img class="d-block w-100" src="img/slider3.jpg" alt="Third slide">
+                <div class="overlay" style="background-image: url('img/overlay.png');height: 100%;position: absolute; top:0;width: 100%;z-index: 2;"></div>
+                <div class="carousel-caption d-none d-md-block" style="z-index: 5">
+                    <h3>QUID PRO QUO</h3>
+                    <p>De tous les peuples de la Gaule, les Belges sont les plus braves</p>
+                </div>
+            </div>
+            <a class="carousel-control-prev mt-8 mb-8" href="#slidebox" role="button" data-slide="prev" style="background-color: red; width:5%; height: 20%; margin-top: 15%; z-index: 3 ">
+              <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+              <span class="sr-only">Previous</span>
+            </a>
+            <a class="carousel-control-next mt-8 mb-8" href="#slidebox" role="button" data-slide="next" style="background-color: red; width:5%; height: 20%; margin-top: 15%;z-index:3;" >
+              <span class="carousel-control-next-icon" aria-hidden="true"></span>
+              <span class="sr-only">Next</span>
+            </a>
+        </div>
+    </div>
+    </ul>
+    </div>
+    <div class="container-fluid bg-danger">
+        <div class="container">
+            <div class="row bg-danger text-white">
+                <div class="col-sm-4 homewidget card bg-danger text-center mb-3 border-0">
```

Comment:
Same here. You could create your own "red" class using CSS and use it in all elements you need :)